runtime/doc/vim_diff.txt

```diff
@@ -69,6 +69,8 @@ are always available and may be used simultaneously in separate plugins.  The
 
 See |nvim-intro| for a list of Nvim's largest new features.
 
+Enables |bracketed-paste-mode|.
+
```

Comment:
This makes it sound as if it's a Neovim feature, but Neovim merely enables and listens to it.